main.js

```diff
@@ -56,27 +57,24 @@ let appShutdownTimeout;
 
 global.indexUrl = 'file://' + __dirname + '/index.html';
 
-
 function openDevTools() {
   childWindow.webContents.openDevTools({ mode: 'undocked' });
   mainWindow.webContents.openDevTools({ mode: 'undocked' });
 }
 
-// Lazy require OBS
-let _obs;
-
-function getObs() {
-  if (!_obs) {
-    _obs = require('obs-studio-node').NodeObs;
-  }
-
-  return _obs;
-}
-
-
 function startApp() {
   const isDevMode = (process.env.NODE_ENV !== 'production') && (process.env.NODE_ENV !== 'test');
 
+  { // Initialize obs-studio-server
+    // Set up environment variables for IPC.
+    process.env.SLOBS_IPC_PATH = "slobs-".concat(uuid());
+    process.env.SLOBS_IPC_WORKING_DIRECTORY = path.join(app.getAppPath().replace('app.asar', 'app.asar.unpacked') + '/node_modules/obs-studio-node');
+    process.env.SLOBS_IPC_USERDATA = app.getPath('userData');
+    // Host a new IPC Server and connect to it.
+    obs.IPC.ConnectOrHost(process.env.SLOBS_IPC_PATH);
+    obs.NodeObs.SetWorkingDirectory(process.env.SLOBS_IPC_WORKING_DIRECTORY);
+  }
```

Comment:
What's with this block?

Reply:
It starts and connects to the IPC server, then sets the proper working directory.